doc/user_guide/user_guide.md

```diff
@@ -590,6 +590,30 @@ INTO SCRIPT CLOUD_STORAGE_EXTENSION.EXPORT_PATH WITH
   PARALLELISM     = 'iproc(), floor(random()*<MULTIPLIER>)';
 ```
 
+### S3 Path Style Access
+
+Amazon S3 [deprecated the path style access][s3-path-style-deprecation] to the
+buckets at the end of the 2020. This breaks the access to the bucket that
+contain dot (`.`) in their names.
+
+[s3-path-style-deprecation]: https://forums.aws.amazon.com/ann.jspa?annID=6776
```

Comment:
This link only works for registered users, maybe one of these links is better?
https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#virtual-hosted-style-access%2F
https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/

Reply:
Fixed.